xep-0402.xml

```diff
@@ -102,25 +115,42 @@
 <conference xmlns=']]>&namespace;<![CDATA['
             name='Council of Oberon'
             autojoin='true'>
-    <nick>Puck</nick>
+  <nick>Puck</nick>
 </conference>
 ]]></example>
         <p>This bookmark would be displayed as 'Council of Oberon' and, if activated, would attempt to join the conference room 'council@conference.underhill.org' with nickname 'Puck'.</p>
         <p>Note that a bookmark item MUST contain only one conference room.</p>
-        <p>Note also that a conference element has no truly mandatory attributes or child elements, though a name SHOULD be given. Thus the following is legal:</p>
+        <p>Note also that a conference element has no truly mandatory attributes or child elements, thus the following is legal:</p>
         <example caption='Minimalist conference element'><![CDATA[
 <conference xmlns=']]>&namespace;<![CDATA['/>
 ]]></example>
     </section1>
 
-    <section1 topic='Examples' anchor='examples'>
+    <section1 topic='Workflow' anchor='workflow'>
+
+        <section2 topic='Registering to notifications' anchor='enable-updates'>
```

Comment:
`Registering to notifications` is grammatically incorrect. You register "for" something, not "to" unless "to" is shorthand for "in order to".

I therefore suggest "Registering to receive notifications".